### PR TITLE
Intial NSXT cloud commit

### PR DESCRIPTION
**Pull request by grastogi23**

### CODE REVIEW

File: examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+#Retrieve vSphere Content Library UUID
+data "vsphere_content_library" "library" {
+  name           = var.content_library_name
+}
+
+#Creates NSX-T User on Avi controller
+resource "avi_cloudconnectoruser" "nsx-t-user" {
+  name           = "nsx-mgr-tf-user"
```
Comment (mayank-avinetworks) [inline examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf]: shall we not read this as well from the variable?

File: examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  tenant_ref     = var.avi_tenant
+  nsxt_credentials {
+    password     = var.nsxt_password
+    username     = var.nsxt_username
+  }
+}
+#Creates vCenter User on Avi controller
+resource "avi_cloudconnectoruser" "vcenter-user" {
+  name           = "vcenter-tf-user"
```
Comment (mayank-avinetworks) [inline examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf]: shall we not read this as well from the variable?

File: examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf (hunk 1 above)
Comment (swapraut) [reply examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf]: I will change this to variable.

File: examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf (hunk 2 above)
Comment (swapraut) [reply examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf]: I will change this to variable.

**Review by mayank-avinetworks — APPROVED** [review]
Approved, assuming the IP addresses in variable.tf files are OK to be exposed.

**Pull request merged**